docs/sources/reference/commandline/cli.md

```diff
@@ -2195,6 +2195,8 @@ available in the default container, you can set these using the `--ulimit` flag.
 > If you do not provide a `hard limit`, the `soft limit` will be used for both
 values. If no `ulimits` are set, they will be inherited from the default `ulimits`
 set on the daemon.
+> `RLIMIT_AS` is disabled for now. In other words, the following script is not supported:
```

Comment:
`RLIMIT_AS` probably doesn't make sense to the end user, and should _probably_ be referred to as `address space` (or `as` as that is the option name).